modules/cstrike/cstrike/CstrikeItemsInfos.cpp

```diff
@@ -190,47 +185,47 @@ bool CsItemInfo::GetAliasInfosFromBuy(const char *alias, AliasInfo *info)
 
 bool CsItemInfo::GetAliasInfosFromName(const char *name, AliasInfo *info)
 {
-	static const char prefix_weapon[] = "weapon_";
-	static const char prefix_item[] = "item_";
+	static const char prefix_weapon[] = "weapon";
+	static const char prefix_item[] = "item";
 
-	const char *alias = name;
+	if (name[6] == '_' && !strncmp(name, prefix_weapon, sizeof(prefix_weapon) - 1))
```

Comment:
Either hardcode the values all the time or use sizeof consistently. Maybe define another const for easier reading?